src/utils/types.ts

```diff
@@ -7,17 +7,10 @@ export type LiteralUnion<T extends U, U = string> =
   | T
   | (U & { zz_IGNORE_ME?: never });
 
-/**
- * Basically a function that returns a value.
- *
- * For some strange reason this is not the same as `Function`.
- */
-export type Callable = (...args) => unknown;
-
 /**
  * Type that represents a single method/function name of the given type.
  */
-export type MethodOf<ObjectType, Signature extends Callable = Callable> = {
+export type MethodOf<ObjectType, Signature extends () => void = () => void> = {
```

Comment:
Maybe

```suggestion
export type MethodOf<ObjectType, Signature extends () => unknown = () => unknown> = {
```

Not tested